### PR TITLE
Add notes to example inventory files about NFS DNS-1123 requirements

### PR DESCRIPTION
**Pull request by csheremeta**
https://bugzilla.redhat.com/show_bug.cgi?id=1703095

### CODE REVIEW

**Comment by vikram-redhat** [comment]
I am not sure how I agree with having the same warning again and again on the same page.

@openshift/team-documentation thoughts?

**Comment by vikram-redhat** [comment]
@openshift/team-documentation *bump*

**Comment by kalexand-rh** [comment]
I think that it's fine to have the note on the same page because they're in different contexts, and the user might not see it if we put it in only the same scenario.

File: install/configuring_inventory_file.adoc [hunk 1]
```diff
@@ -1208,6 +1208,9 @@ be *_/exports/registry_*:
 ----
 [OSEv3:vars]
 
+# nfs_directory must conform to DNS-1123 subdomain must consist of lower case
```
Comment (kalexand-rh) [inline install/configuring_inventory_file.adoc]: Maybe:
> The `openshift_hosted_registry_storage_nfs_directory` must match the DNS-1123 subdomain name. Its name must consist of lower case lower case alphanumeric characters, dashes ('-)', or periods ('.') and must start and end with an alphanumeric character

**Comment by vikram-redhat** [comment]
I am merging this. If any changes need to be made for the language, they can come at a later time.

**Pull request merged**